### ISSUE TITLE
New GSI fix set for GFS v16.3.12

### ISSUE THREAD

**Issue by ADCollard** [NOAA-EMC/global-workflow#2282]
### Target fix directory

gsi

### Type of change

- [ ] Update existing files
- [ ] Remove files
- [ ] Add new files to existing directory
- [x] Add new fix directory

### Any workflow changes needed?

No change needed

### Related issues

https://github.com/NOAA-EMC/GSI-fix/pull/13

### Pickup location

/scratch1/NCEPDEV/da/Andrew.Collard/git/GSI_gfsda_v16/fix/Rcov_crisn21 & GSI-fix repo clone

### Size

Full new copy of GSI fix subset

**Comment by KateFriedman-NOAA** [comment]
@ADCollard I can certainly add this new file into the current `fix/gsi/20230911` timestamp version within the developmental fix set. Are the updated GFSv16 `global_anavinfo.l127.txt`, `global_convinfo.txt`, and `global_satinfo.txt` files coming later? I would create a new GSI timestamp set for them when they do.

**Comment by KateFriedman-NOAA** [comment]
Is this a duplicate of issue #1919?

**Comment by RussTreadon-NOAA** [comment]
@KateFriedman-NOAA , you are right.  #1919 is no longer needed.

**Comment by KateFriedman-NOAA** [comment]
I have copied the new `Rcov_crisn21` file into the GSI `20230911` timestamp subfolder:

```
[role.glopara@hfe05 20230911]$ pwd
/scratch1/NCEPDEV/global/glopara/fix/gsi/20230911
[role.glopara@hfe05 20230911]$ ll Rcov_crisn21
-rw-r--r-- 1 role.glopara global 40436 Feb  1 16:33 Rcov_crisn21
```

**Comment by ADCollard** [comment]
@KateFriedman-NOAA Thanks!

The other files are included in the [GSI-Fix PR13](https://github.com/NOAA-EMC/GSI-fix/pull/13).  Do I need to explicitly add then to this issue?

(Apologies, I cannot assign any one in the GSI-Fix repositorty, but I will go ahead and tag people in the commentary)

**Comment by KateFriedman-NOAA** [comment]
@ADCollard Previously, a DA team member provided me with a copy of the newest GSI-fix on Hera and I copied it into a new timestamp subfolder of `fix/gsi` within the developmental fix set (and synced to the other platform copies). Once https://github.com/NOAA-EMC/GSI-fix/pull/13 is in, I would expect to get a request to add a new timestamp copy and populate it with the updated GSI-fix hash. Then I would update `gsi_ver` in `versions/fix.ver` in global-workflow in the same branch that would collect the related script updates that go with the fix updates (issue #1918). Since I know the `global_satinfo.txt` and two other `global_*.txt` files are changing compared to their current versions in the latest GSI fix timestamp, I need to make a whole new timestamp copy to preserve the existing set. I was able to add that new `Rcov_crisn21` fix file to the current timestamp copy since it didn't already exist. Hope that makes sense. Would handing off the last GSI-fix hash after that PR goes in work for you for the fix updates?

Also, please see the checklist in issue #1918 for the updates I am expecting to make in global-workflow `develop` to bring in recent GFSv16.3.* changes. Let me know in that issue if any of those checklist items won't be happening in that effort. Thanks!

**Comment by ADCollard** [comment]
@KateFriedman-NOAA [NOAA-EMC/GSI-fix#13](https://github.com/NOAA-EMC/GSI-fix/pull/13) has now been merged into develop.  Can we no go head and produce a new timestamp copy?

Thanks,
Andrew.

**Comment by KateFriedman-NOAA** [comment]
@ADCollard Certainly! I also see that https://github.com/NOAA-EMC/GSI-fix/pull/14 was merged into GSI-fix develop. Do you want that update included (new hash [298bdc0](https://github.com/NOAA-EMC/GSI-fix/commit/298bdc0f56692ad25141a946177e5d88884e367a)) or just the state of GSI-fix after https://github.com/NOAA-EMC/GSI-fix/pull/13 went in (new hash https://github.com/NOAA-EMC/GSI-fix/commit/cd0a90cd4a0e4eba4c1b8105ebc0a1c952d9764b)?

**Comment by ADCollard** [comment]
@KateFriedman-NOAA Yes, please proceed with both updates.

**Comment by KateFriedman-NOAA** [comment]
@ADCollard I have cloned the HEAD of GSI-fix into a new `20240208` timestamp subfolder. See here on Hera: `/scratch1/NCEPDEV/global/glopara/fix/gsi/20240208`. It looks like it's missing some files, in particular `Rcov_crisn21` and other `Rcov_*` files. How do you suggest we collect this new timestamp copy with all files? Previously I think someone provided me with a staged complete copy to pull in. Perhaps that would be best this time?

**Comment by ADCollard** [comment]
@KateFriedman-NOAA Thanks for doing this.  I think the cleanest way is to copy the binary files from the previous timestamp folder.

**Comment by KateFriedman-NOAA** [comment]
@ADCollard Alrighty...I removed the `20240208` copy I'd cloned, copied the `20230911` folder as `20240208`, and then did a `git pull` in the `20240208` folder to pull in the GSI-fix repo updates. Here are the diffs that got pulled in:

```
[role.glopara@hfe08 20240208]$ git pull
remote: Enumerating objects: 9, done.
remote: Counting objects: 100% (9/9), done.
remote: Compressing objects: 100% (8/8), done.
remote: Total 9 (delta 3), reused 5 (delta 1), pack-reused 0
Unpacking objects: 100% (9/9), done.
From https://github.com/NOAA-EMC/GSI-fix
   5722cd4..298bdc0  develop    -> origin/develop
Updating 5722cd4..298bdc0
Fast-forward
 global_anavinfo.l127.txt          |   6 +-
 global_anavinfo_soilanal.l127.txt | 137 ++++++++++++++++++++++++++++
 global_convinfo.txt               |   2 +-
 global_convinfo_2mObs.txt         | 262 +++++++++++++++++++++++++++++++++++++++++++++++++++++
 global_satinfo.txt                | 470 +++++++++++++++++++++++++++++++++++++++++++++++------------------------------------------------
 5 files changed, 637 insertions(+), 240 deletions(-)
 create mode 100644 global_anavinfo_soilanal.l127.txt
 create mode 100755 global_convinfo_2mObs.txt
```

Please take a look at this new `/scratch1/NCEPDEV/global/glopara/fix/gsi/20240208` GSI fix subfolder on Hera and let me know if it's good-to-go. I'm guessing that there may be files in the `20240208` copy that no longer exist and could be removed....please let me know if so and which files. I'd like to keep the `20240204` set as clean as possible. Thanks!

**Comment by ADCollard** [comment]
@KateFriedman-NOAA The contents of `/scratch1/NCEPDEV/global/glopara/fix/gsi/20240208` are as expected.   I do not think there are any files that do not exist, but there is probably scope to remove obsolete files.   But that is not something that I would be comfortable doing unilaterally - we probably need the DA team as a whole to review.

**Comment by KateFriedman-NOAA** [comment]
@ADCollard Gotcha. I will proceed with syncing the new `fix/gsi/20240208` version to the other supported platforms and then close this issue as complete. Make sure to update `gsi_ver=20240208` in your branch for issue #1918. I will add another checklist item to that issue as a reminder for us.

**Comment by KateFriedman-NOAA** [comment]
@WalterKolczynski-NOAA If you haven't already since yesterday, please sync the fix set on Orion to pull in the new `20240208` GSI subset. Thanks!

**Comment by KateFriedman-NOAA** [comment]
New GSI fix set (`20240208`) has now been synced to all supported platforms. Closing issue as complete.

The `gsi_ver` in `versions/fix.ver` will be taken care of in issue #1918.
